public/js/list.class.ts

```diff
@@ -0,0 +1,112 @@
+export class List{
```

Comment:
Missing space after List. Have you considered using a formatter such as [prettier](https://prettier.io)?

Reply:
I didn't use prettier ever. I will test it! Thanks!